src/script/format/chemGraph/toGraph/prepare.js

```diff
@@ -40,17 +40,16 @@ export function prepareStructForGraph(struct) {
   const molFrags = Array.from(struct.frags.keys()).filter(
     fid => !rgFrags.has(fid)
   )
-  const moleculeFragsAtoms = molFrags.reduce((acc, fid) => {
-    const fragAtoms = struct.getFragmentIds(fid)
-    acc = acc.union(fragAtoms)
-    return acc
-  }, new Pile())
+  for (const fid of molFrags) {
```

Comment:
Please revert this changes: it is necessary to save structure as a single molecule